src/test/java/org/mule/extension/db/integration/storedprocedure/StoredProcedureTestCase.java

```diff
@@ -78,4 +87,13 @@ public void outParam() throws Exception {
     assertThat("3", equalTo(payload.get("count").toString()));
   }
 
+  @Test
+  public void runStoredProcedureWithArgumentThatDoesNotExists() {
+    try {
+      runProcedure("callNotExistingStoredProcedureWithAnArgument");
+    } catch (Exception e) {
+      assertThat(e.getCause().getCause(), is(instanceOf(IllegalArgumentException.class)));
```

Comment:
after changing the exception, this validation should change to validate not only the exception but also the Mule Error